src/components/InputBook.js

```diff
@@ -18,29 +20,34 @@ export const InputBook = () => {
     });
   };
 
-  const handleAddBook = (e) => {
+  const handleAddBook = async (e) => {
     e.preventDefault();
     if (inputBook.title.trim()) {
+      const bookObj = {
+        item_id: uuidv4(),
+        title: inputBook.title,
+        author: inputBook.author,
+        category: inputBook.category,
+      };
+
+      await dispatch(addBookEvent(bookObj));
+      await dispatch(LoadBooks());
       setInputBook({
         title: '',
         author: '',
+        category: '',
       });
-      dispatch(
-        addBookEvent({
-          title: inputBook.title,
-          author: inputBook.author,
-        }),
-      );
     } else {
       // eslint-disable-next-line no-alert
       alert('Please write a title');
     }
   };
 
   return (
-    <div>
+    <div className="div-form">
+      <hr className="hr-line" />
+      <h2 className="form-header"> Add a New Book</h2>
```

Comment:
- [ ] Well done for implementing the horizontal border. However, let's try to ensure that we use the correct specified color for the border. Please check the zeplin design 👍 
- [ ]  As stated earlier please align the Add a New Book heading to the left as shown in the given design\
### Your design
![image](https://user-images.githubusercontent.com/46839548/208254967-f0169269-8ff9-4045-9c9b-5d490f5ec4f3.png)
### Zeplin design
![image](https://user-images.githubusercontent.com/46839548/208255019-404f11d0-cdc3-4e45-a29f-bd59584ad38d.png)